src/sources/kubernetes_logs/mod.rs

```diff
@@ -453,30 +456,30 @@ impl Source {
         let event_processing_loop = out.send_event_stream(&mut stream);
 
         let mut lifecycle = Lifecycle::new();
-        {
-            let (slot, shutdown) = lifecycle.add();
-            let fut =
-                util::cancel_on_signal(reflector_process, shutdown).map(|result| match result {
-                    Ok(()) => info!(message = "Reflector process completed gracefully."),
-                    Err(error) => emit!(&KubernetesLifecycleError {
-                        error,
-                        message: "Reflector process exited with an error."
-                    }),
-                });
-            slot.bind(Box::pin(fut));
-        }
-        {
-            let (slot, shutdown) = lifecycle.add();
-            let fut =
-                util::cancel_on_signal(ns_reflector_process, shutdown).map(|result| match result {
-                    Ok(()) => info!(message = "Namespace reflector process completed gracefully."),
-                    Err(error) => emit!(&KubernetesLifecycleError {
-                        error,
-                        message: "Namespace reflector process exited with an error.",
-                    }),
-                });
-            slot.bind(Box::pin(fut));
-        }
+        // {
```

Comment:
TODO: it seems totally fine operationally to leave these out, I haven't seen any impacts but...